public/locales/en/translation.json

```diff
@@ -13,5 +13,26 @@
   },
   "ar": {
     "1": "Arabic"
+  },
+  "whoAreWe": {
+    "header": "who are we?"
+  },
+  "aboutPart": {
+    "0": "We are",
```

Comment:
there are very, very very few cases where you should be using "0", "1", "2", "3" as names

please give these each meaningful names based on their content

example, 0 is the header

i leave to you to to name the other 3